website/client/components/challenges/challengeModal.vue

```diff
@@ -228,11 +234,11 @@ export default {
     }
 
     this.groups.push({
-      name: 'Public',
+      name: 'Public Challenges',
```

Comment:
Ah, thanks, yes, I should have noticed that! :) 

How do I do translatable text within a <script> section in a vue file? I've tried a quick search for other examples but can't find any and I have to get ready for work soon so I can't keep searching. :(